go/oasis-test-runner/scenario/e2e/stake_cli.go

```diff
@@ -414,6 +429,19 @@ func (s *stakeCLIImpl) checkEscrowBalance(childEnv *env.Env, src signature.Publi
 	return nil
 }
 
+func (s *stakeCLIImpl) showTx(childEnv *env.Env, txPath string) error {
+	s.logger.Info("pretty printing generated transaction")
+
+	args := []string{
+		"consensus", "show_tx",
+		"--" + consensus.CfgTxFile, txPath,
+	}
+	if err := runSubCommand(childEnv, "show_tx", s.basicImpl.net.Config().NodeBinary, args); err != nil {
```

Comment:
why is subcommand the preferred way to do this

Reply:
Adding the subcommand was response to a review suggestion from @Yawning. I think it is useful to have a subcommand for displaying the content of transactions.